### PR TITLE
PLAT-611 | Disable instalments for album and remove note about access in invoice for album's instalment

### PR DESCRIPTION
**Pull request by bognix**
When releasing remember to **remove album's instalments row from payment_method_product table**

### CODE REVIEW

File: resources/assets/js/components/user/Order.vue [hunk 1]
```diff
@@ -218,7 +218,7 @@
 						<a class=""
 							title="Dodaj lub zmień kod rabatowy"
 							@click="toggleCouponInput"
-							v-if="order.status !== 'closed'">
+							v-if="order.status !== 'clo	sed'">
```
Comment (gitkuba) [inline resources/assets/js/components/user/Order.vue]: o so chosi?

File: lib/Invoice/Invoice.php [hunk 2]
```diff
@@ -587,6 +587,10 @@ private function price($number)
 
 	private function getInstalmentsNote($order)
 	{
+		if (empty($order->product->access_start) || empty($order->product->access_end)) {
+			return '';
```
Comment (gitkuba) [inline lib/Invoice/Invoice.php]: Nie wiem co o tym myśleć, może to przegadajmy po prostu.

File: resources/assets/js/components/user/Order.vue (hunk 1 above)
Comment (bognix) [inline resources/assets/js/components/user/Order.vue]: ay ay

File: lib/Invoice/Invoice.php (hunk 2 above)
Comment (bognix) [reply lib/Invoice/Invoice.php]: discussed offline, couldn't think of anything better for now :(

**Pull request merged**